components/dashboard/src/AppNotifications.tsx

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { useCallback, useEffect, useState } from "react";
+import Alert, { AlertType } from "./components/Alert";
+import { useFeatureFlag } from "./data/featureflag-query";
+import { LinkButton } from "./components/LinkButton";
+
+const KEY_APP_DISMISSED_NOTIFICATIONS = "gitpod-app-notifications-dismissed";
+
+interface Notification {
+    id: string;
+    type: AlertType;
+    message: JSX.Element;
+}
+
+const SCHEDULED_DOWNTIME: Notification = {
+    id: "230924-scheduled-downtime",
+    type: "info",
+    message: (
+        <span className="text-md">
+            To improve the reliability of gitpod.io we are performing a scheduled maintenance this{" "}
+            <span className="font-semibold">Sunday, Sept. 24th, 05:00-06:00 UTC</span>.{" "}
+            <LinkButton onClick={() => window.open("https://www.gitpodstatus.com/incidents/0d5vlgxcp27v", "_blank")}>
```

Comment:
Could we just use a normal `<a>` tag here since we just open a link?

Reply:
I tried that, but then it gets rendered as regular text for some reason. :shrug: 

Would you have time to pair on this, maybe? :orange_heart: